application_form/tests/test_export_service.py

```diff
@@ -210,8 +210,8 @@ def test_export_sale_report(
         )
         reservation.set_state(ApartmentReservationState.SOLD)
 
-    start_date = timezone.localdate() - timedelta(days=7)
-    end_date = timezone.localdate() + timedelta(days=7)
+    start_date = timezone.now() - timedelta(days=7)
+    end_date = timezone.now() + timedelta(days=7)
```

Comment:
I guess it would be better to not call these `date`s anymore as they are actually datetimes now.

Reply:
I'll fix that...

Reply:
Fixed.